src/test/resources/real-world-gtfs-feeds/VTA-gtfs-multiple-trips/stop_times.txt

```diff
@@ -23,4 +23,6 @@ trip_id,arrival_time,departure_time,stop_id,stop_sequence,stop_headsign,pickup_t
 5,23:59:00,23:59:00,1562,3,,0,0,9.39290047,0
 6,00:00:00,00:00:00,4957,1,,0,0,,1
 6,,,1558,2,,0,0,8.34879971,0
-6,23:59:00,23:59:00,1562,3,,0,0,9.39290047,0
\ No newline at end of file
+6,23:59:00,23:59:00,1562,3,,0,0,9.39290047,0
+7,18000,18000,4957,1,,0,0,,0
+7,18000,18000,1558,2,,0,0,8.34879971,0
```

Comment:
Are columns 2 and 3 at line 7 supposed to be stop times?

Reply:
The feed data provided to replicate the data had this format for arrival and departure times. Changing the format in the test data to HH:MM:SS fixed the failing tests.